test/controllers/api/v0/imports_controller_test.rb

```diff
@@ -64,9 +64,10 @@ def teardown
 
     pages = Page.where(url: 'http://testsite.com/')
     assert_equal 1, pages.length
-    assert_equal import_data[0][:page_title], pages[0].title
     assert_equal import_data[0][:site_agency], pages[0].agency
     assert_equal import_data[0][:site_name], pages[0].site
+    assert_equal import_data[0][:page_title], pages[0].latest.title
+    assert_nil pages[0].title
```

Comment:
Shouldn’t this *not* be `nil`? i.e. the earlier assertion (`assert_equal import_data[0][:page_title], pages[0].title`) should still be here, shouldn’t it?

Reply:
We should probably also change the import data here to `title` instead of `page_title`, or have tests that use both (since we’re transitioning from one to the other).

Reply:
Yes, it makes sense that this test should still pass as is. I dug a little more into this, and it seems that [this line](https://github.com/edgi-govdata-archiving/web-monitoring-db/blob/master/app/jobs/import_versions_job.rb#L69) wasn't actually triggering the `after_add` callback on the `Page` model. Interestingly enough, if I move the callback into the `Version` model in the form of an `after_save`, it does get triggered, so I'll push up that change as well.

Reply:
I'll also revert the test back. I should have trusted the test as it was originally written.

Reply:
Oh, I think the more immediate problem was actually https://github.com/edgi-govdata-archiving/web-monitoring-db/blob/master/app/jobs/import_versions_job.rb#L52, which should have been `page.versions.push(version)`

Reply:
Ah, yep I think you're right!